configuration/concepts/1340_index_testing_program.csv

```diff
@@ -0,0 +1,18 @@
+Uuid,Members,Answers,Fully specified name:fr,Fully specified name:en,Short name:fr,Short name:en,Data class,Data type,_order:1340
+37b12af0-5231-11ea-8d77-2e728ce88125,,,Index Testing Program,Index Testing Program,Index Testing Program,Index Testing Program,Misc,Text,
+6bcd71bc-5232-11ea-8d77-2e728ce88125,,,Refusé le dépistage,Refused Testing,Refusé le dépistage,Refused Testing,Misc,N/A,
+6bcd6f00-5232-11ea-8d77-2e728ce88125,,,Accord pour le dépistage,Accepted Testing,Accord pour le dépistage,Accepted Testing,Misc,N/A,
+6bcd7522-5232-11ea-8d77-2e728ce88125,Refused Testing;Accepted Testing,,Index Testing Résultats du programme,Index Testing Program Outcomes,Résultats du programme,Program Outcomes,Misc,Coded,
```

Comment:
@lauravignoli "Index Testing Résultats du programme" can be translated to "Résultats du Programme Cas Index"